osmapi/OsmApi.py

```diff
@@ -1963,33 +1973,33 @@ def _http(self, cmd, path, auth, send):  # noqa
                 self._conn = self._get_http_connection()
```

Comment:
@metaodi I agree that renaming the variable to `_session` provides some additional clarity because it is reusing the same HTTP session.

I changed the naming in my latest two commits.